src/LaunchDarkly.Logging.Microsoft/LaunchDarkly.Logging.Microsoft.csproj

```diff
@@ -1,25 +1,25 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>3.0.0</Version>
+    <Version>3.1.0</Version>
     <!--
       The reason there's a mechanism here for overriding the target frameworks with
       an environment variable is that we want to be able to run CI tests using older
       SDKs which do not consider "net5.0" to be a valid target framework that can be
       referenced in a project file.
     -->
-    <BuildFrameworks Condition="'$(BUILDFRAMEWORKS)' == ''">netstandard2.0;netstandard2.1</BuildFrameworks>
+    <BuildFrameworks Condition="'$(BUILDFRAMEWORKS)' == ''">netstandard2.0;netstandard2.1;net6.0;net7.0</BuildFrameworks>
```

Comment:
We aren't targeting any specific feature of .Net 6.0 or 7.0, so a build of netstandard should work for both those use cases.

Reply:
https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1